setup.py

```diff
@@ -22,7 +22,6 @@
         "sqlalchemy-utils>=0.32,<0.36.4",
         "psycopg2~=2.7",
         "cdislogging~=1.0",
-        "indexclient @ git+https://github.com/NCI-GDC/indexclient.git@2.0.0#egg=indexclient",
```

Comment:
Because setup.py's `install_requires` section changed we should re-run `pip-compile` so the changes get reflected there as well.